grouper/models.py

```diff
@@ -118,6 +130,17 @@ def get_db_engine(url):
     return create_engine(url, pool_recycle=300)
 
 
+Plugins = []
+
+
+def load_plugins(plugin_dir):
+    """Load plugins from a directory"""
+    global Plugins
+    if Plugins:
+        raise Exception("Plugins already loaded; can't load twice!")
```

Comment:
I feel like this should have its own exception. Something like:

```python
raise PluginsAlreadyLoaded("Plugins already loaded; can't load twice!")
```

That way you can catch that exception? Base `Exception` can be hard to isolate.